### PR TITLE
NXP-28610: ensure tasks drawer is updated when task is processed

### PR DESCRIPTION
**Pull request by nmpcunha**

### CODE REVIEW

File: elements/nuxeo-app.js [hunk 1]
```diff
@@ -912,6 +912,8 @@ Polymer({
     // let's refresh the current document since it might have been changed (ex: state and version)
     this.fire('document-updated');
     this._fetchTaskCount();
+    this.currentTask = null;
+    this.currentTaskId = null;
```
Comment (Gabez0r) [inline elements/nuxeo-app.js]: note: task selection will be lost here, which I think is ok, since it is in fact a different task

Comment (Gabez0r) [inline elements/nuxeo-app.js]: I don't think resetting the current task is the key here. The main problem is that the drawer is not updated when open, not only when you process a task (addressed by these lines), but also when you start or end a workflow. I'd expect us to actually refetch the tasks, in order to receive any changes that might have occurred; something along these lines:
```
    const tasksDrawer = this.$$('nuxeo-tasks-drawer');
    if (tasksDrawer.visible) {
      tasksDrawer.$.tasks.fetch();
    }
```

Comment (nmpcunha) [reply elements/nuxeo-app.js]: Yes, that was the aim. 👍

Comment (nmpcunha) [reply elements/nuxeo-app.js]: Yes, probably it is safer to have that approach for all tasks events that follow this flow. 
Thanks!

File: elements/nuxeo-app.js [hunk 2]
```diff
@@ -615,6 +615,11 @@ Polymer({
     Performance.mark('nuxeo-app.ready');
   },
 
+  _resetSelections() {
```
Comment (Gabez0r) [inline elements/nuxeo-app.js]: nit: I'd name this `_resetSelection` (singular) or even `_resetTaskSelection`, since we're only changing task selection.

Comment (nmpcunha) [reply elements/nuxeo-app.js]: Fair! 👍

**Pull request merged**